modules/cli/pages/cli-intro.adoc

```diff
@@ -39,46 +39,117 @@ You can collect diagnostics through the command-line interface by using the xref
 [#server-tools-packages]
 == Server Tools Packages
 
-For convenience, Couchbase provides a  tools package  that contains the following utilities:
+For convenience, Couchbase provides Server developer tools package and Server admin tools package.
+
+=== Server Developer Tools Packages
+
+Server developer tools package contains the following utilities:
+
+* xref:tools:cbimport.adoc[`cbimport`]
+* xref:tools:cbexport.adoc[`cbexport`]
+* xref:cli:cbq-tool.adoc[`cbq`]
+* xref:backup-restore:cbbackupmgr.adoc[`cbbackupmgr`]
+* xref:cbcli:couchbase-cli.adoc[`couchbase-cli`]
+* cbstats
+* mctimings
+* mcstat
+
+This package lets you install the following Server developer tools on systems where you have not installed Couchbase Server.
```

Comment:
It says "following Server developer tools" ... so, maybe this sentence needs to be moved to the top before the list of utilities.